.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
     - DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     - DOTNET_CLI_TELEMETRY_OPTOUT: 1
     - SIGNALR_TESTS_VERBOSE: 1
-    - TRAVIS_NODE_VERSION: 6.9.1
+    - TRAVIS_NODE_VERSION: 8.0.0
```

Comment:
Does this work? We currently have this warning: 
`npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.`

https://github.com/gulpjs/gulp/issues/1571

I tried to fix it but it is a dependency that we don't control.